### ISSUE TITLE
Add support for theme_boost

### ISSUE THREAD

**Issue by abias** [itamart/moodle-local_loginas#9]
Hi,

in theme_boost in Moodle 3.2, there is no settings block anymore and thus local_loginas does not work here anymore.

Do you have any plans to support theme_boost, for example by adding local_loginas to the nav drawer where the site administration link is already placed?

Thanks,
Alex

**Comment by ghost** [comment]
Yeah, this is a huge issue as Boost is a default theme in Moodle 3.2. When will this be fixed?
